docs/installing-solidity.rst

```diff
@@ -115,15 +115,14 @@ they relate only to Alethzero and Mix
     git clone --recursive https://github.com/ethereum/webthree-umbrella.git
     cd webthree-umbrella
     ./webthree-helpers/scripts/ethupdate.sh --no-push --simple-pull --project solidity # update Solidity repo
-    ./webthree-helpers/scripts/ethbuild.sh --no-git --project solidity --all --cores 4 # build Solidity
+    ./webthree-helpers/scripts/ethbuild.sh --no-git --project solidity --cores 4 -DEVMJIT=0 # build Solidity
 
 If you opted to install Alethzero and Mix:
 
 .. code-block:: bash
 
-    git clone --recursive https://github.com/ethereum/webthree-umbrella.git
-    cd webthree-umbrella && mkdir -p build && cd build
-    cmake ..
+    ./webthree-helpers/scripts/ethupdate.sh --no-push --simple-pull --project all # update Solidity repo
```

Comment:
I haven't tried the cmake route...either way we need to specify that for OS X that they need to make sure that they set DEVMJIT to 0.